packages/test/test-end-to-end-tests/src/test/batchBreakRegression.spec.ts

```diff
@@ -377,7 +378,8 @@ describeNoCompat("Batching failures", (getTestObjectProvider) => {
 					assert.fail("expected error");
 				} catch (e) {
 					assert(isILoggingError(e), `${e}`);
-					assert(e instanceof DataProcessingError);
+					assert(isFluidError(e));
+					assert(e.errorType === FluidErrorTypes.dataProcessingError);
```

Comment:
For better assertion error messages? Same below.
```suggestion
					assert.strictEqual(e.errorType, FluidErrorTypes.dataProcessingError);
```

Reply:
Good catch. Thanks!